jobs/build-arch.Jenkinsfile

```diff
@@ -139,6 +142,7 @@ lock(resource: "build-${params.STREAM}-${params.ARCH}") {
         """)
 
         // declare these early so we can use them in `finally` block
+        assert params.VERSION != ""
```

Comment:
Is there no way to mark a parameter as required when declaring the parameter above?

Reply:
Not AFAIK.